downstream/assemblies/eda/assembly-eda-projects.adoc

```diff
@@ -2,8 +2,9 @@
 
 = Projects
 
-[role="_abstract"]
+Projects are a logical collection of rulebooks. They must be a git repository and only http protocol is supported. 
```

Comment:
Hi, I just noticed that there is an important lack here. Projects must meet an specific layout. I was going to create a bug for it, but if we are updateing the section here, we can add it in the PR. What do you think @EMcWhinn ?

Reply:
It would be just a sentence like: 
"The rulebooks of a project must be located at the `/rulebooks` folder in the root of the project or at the path defined for eda content in ansible collections: `/extensions/eda/rulebooks`